definitions/npm/prismic.io_v3.x.x/flow_>=v0.28.x/prismic.io_v3.x.x.js

```diff
@@ -1,5 +1,5 @@
 declare module 'prismic.io' {
-  declare type linkResolver = (...args: any) => any;
+  declare type LinkResolver = (...args: any) => any;
```

Comment:
@jeffmo Technically this is a breaking change because it changes the name of the type which could be imported. Do we want to allow these breaking changes? Do we want some way of versioning them?